src/VkBootstrap.cpp

```diff
@@ -1633,22 +1633,29 @@ Result<SurfaceSupportDetails> query_surface_support_details(VkPhysicalDevice phy
 	return SurfaceSupportDetails{ capabilities, formats, present_modes };
 }
 
-VkSurfaceFormatKHR find_surface_format(VkPhysicalDevice phys_device,
+Result<VkSurfaceFormatKHR> find_desired_surface_format(VkPhysicalDevice /* phys_device */,
     std::vector<VkSurfaceFormatKHR> const& available_formats,
```

Comment:
Should remove the phys_device parameter if it is unused.

Reply:
Should be resolved. Same as below.